stylesheets/_structure.forms.scss

```diff
@@ -54,7 +54,7 @@
 
 // Add a gutter between form labels and controls
 .control__label + .controls {
-    @include col-span($form-control-cols, false, true);
+    @include col-span($form-control-cols, false);
```

Comment:
Won't this remove the gutter from all controls?

Reply:
Not that I've noticed

Reply:
I'd double check, that 3rd param is gutter and it's defaults to false.